apartment_application_service/settings.py

```diff
@@ -176,8 +176,35 @@
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "handlers": {"console": {"class": "logging.StreamHandler"}},
-    "loggers": {"django": {"handlers": ["console"], "level": env.str("LOG_LEVEL")}},
+    "formatters": {
+        "standard": {
```

Comment:
Is this real standard? If this not has any real standard the name should change to other. This can mislead.

Reply:
This is thought as standard relative to this application. Name can be changed.

Reply:
How about `Verbose` as it gives much more information?